Tlaka95/COS301_TECH_TITANS_TEAM
# Skeleton SRS Template

@Oratile-15306195  Please review this and approve if all is good

Steps to review
1. go to the SkeletonSRS_Template branch
2. download the branch
3. compile the SRS document on your local machine 
4. if it compiles successfully
  4.1 check for structure and grammar errors
  4.2  come back to git then approve and I'll merge the request after you merge
